greengrass_parsec_workshop/build_demo.sh

```diff
@@ -12,11 +12,13 @@ if test -e /etc/hostname; then
 fi
```

Comment:
This would fail if hostname contains dots. GG think name can't contains dots, a full hostname might